packages/edit-navigation/src/components/layout/style.scss

```diff
@@ -39,6 +39,25 @@
 		// deselection to work.
 		flex-grow: 1;
 	}
+
+	.interface-interface-skeleton__header {
+		border-bottom-color: transparent;
+	}
+
+	// Force the sidebar to the right side of the screen on larger
+	// breakpoints.
+	&.has-permanent-sidebar .interface-interface-skeleton__sidebar {
+		position: fixed !important;
```

Comment:
is there any way to avoid using !important?

Reply:
Looks like this package is using `important` here:
https://github.com/WordPress/gutenberg/blob/1c2c0aed196810beb1c9527f9fa0a77b2efe2fcb/packages/interface/src/components/interface-skeleton/style.scss#L105

This might be a bit fragile if CSS loading order changes. With multiple `!important` rules, last one read wins.